src/encoded/schemas/experiment_mic.json

```diff
@@ -84,7 +84,7 @@
                     },
                     "channel":{
                         "title": "Imaging channnel",
-                        "description" : "channel info, ie. ch01, ch02...",
+                        "description" : "channel info, ie. ch00, ch01...",
```

Comment:
With the changes and what you have said about there really only being a max of 4 channels used - should this be restricted to an enum or suggested_enum - or do you think people will go ahead and build >4 channel scopes just to spite us if we did?

Reply:
It's actually up to 5 channels and it makes sense to have this as enum. If there will ever be more than 5 channels, then more things need to be changed (i.e. MicroscopeSettings), so at that point we can revise also this enum.